File: Codechef/Odd_Sum_Pair/Odd_Sum_Pair.cpp
#include <iostream>
using namespace std;

int main()
{
    int T;
    cin >> T;
    while (T--)
    {
        int A, B, C;
        cin >> A >> B >> C;
        if (((A + B) %2 == 1) || ((A + C) % 2 == 1) || ((B + C) % 2 == 1))
        {
            cout << "YES" << endl;
        }
        else
        {
            cout << "NO" << endl;
        }
    }
    return 0;
}

File: Codechef/Odd_Sum_Pair/Odd_Sum_Pair.py
T = int(input())
while T:
    T -= 1
    A, B, C = map(int, input().split())
    if (((A + B) % 2 == 1) or ((A + C) % 2 == 1) or ((B + C) % 2 == 1)):
        print("YES")
    else:
        print("NO")
